### PR TITLE
data/50unattended-upgrades.Ubuntu: add new ESM repositories

### PR DESCRIPTION
**Pull request by rbalint**
LP: #1857051

### CODE REVIEW

File: data/50unattended-upgrades.Ubuntu [hunk 1]
```diff
@@ -11,6 +11,8 @@ Unattended-Upgrade::Allowed-Origins {
 	// available, the policy for updates is such that unattended-upgrades
 	// should also install from here by default.
 	"${distro_id}ESM:${distro_codename}";
+	"${distro_id}ESM:${distro_codename}-apps-security";
```
Comment (blackboxsw) [inline data/50unattended-upgrades.Ubuntu]: Thanks for tackling this @rbalint, my fault on the bug file typo. ESM for apps has a different origin (UbuntuESMApps) than ESM for infra (UbuntuESM).

I have updated the bug description and title.
https://bugs.launchpad.net/ubuntu/+source/unattended-upgrades/+bug/1857051 

So I think this line should be
"${distro_id}ESMApps:${distro_codename}-apps-security";

File: data/50unattended-upgrades.Ubuntu [hunk 2]
```diff
@@ -11,6 +11,8 @@ Unattended-Upgrade::Allowed-Origins {
 	// available, the policy for updates is such that unattended-upgrades
 	// should also install from here by default.
 	"${distro_id}ESM:${distro_codename}";
```
Comment (blackboxsw) [inline data/50unattended-upgrades.Ubuntu]: Please do drop the old (and now unmatching) "${distro_id}ESM:${distro_codename}"

File: data/50unattended-upgrades.Ubuntu [hunk 3]
```diff
@@ -11,6 +11,8 @@ Unattended-Upgrade::Allowed-Origins {
...
+	"${distro_id}ESM:${distro_codename}-apps-security";
+	"${distro_id}ESM:${distro_codename}-infra-security";
 	"${distro_id}ESM:${distro_codename}-security";
```
Comment (blackboxsw) [inline data/50unattended-upgrades.Ubuntu]: We can also drop this line too as no ESM origins will publish this origin.

**Comment by frankban** [comment]
Hey @blackboxsw , I confirm UbuntuESM for infra and UbuntuESMApps for apps. Could you expand a bit on the second question?

**Comment by blackboxsw** [comment]
@frankban the existing unattended=upgrades config surfaces the following additional two 

'Allowed-Origins' for auto-updates:
"${distro_id}ESM:${distro_codename}";
AND
"${distro_id}ESM:${distro_codename}-security";

This config also allows for unattended updates from the following origin/pockets:

UbuntuESM:trusty
AND
UbuntuESM:trusty-security 

I am suggesting that those two items also be dropped from default unattended upgrades config as I thought ESM repos would not actually be surfacing any packages in **origin=UbuntuESM**: suites: **<series>-security** or **<series>**.

I *think* those are legacy origins and suite/pocket names that do not exist anymore right?


https://github.com/mvo5/unattended-upgrades/pull/243/files#r367671165

**Comment by frankban** [comment]
@blackboxsw trusty does not exist, trusty security exists for backward compatibility, but they can be dropped yes in favor of trusty-infra-security and trusty-infra-updates.

**Comment by rbalint** [comment]
Reopened as https://github.com/mvo5/unattended-upgrades/pull/244

**Pull request closed without merging**